includes/express-checkout/class-wc-payments-express-checkout-button-handler.php

```diff
@@ -446,4 +452,23 @@ public function filter_gateway_title( $title, $id ) {
 
 		return $title;
 	}
+
+	/**
+	 * Add ECE data to `wcSettings` to allow it to be accessed by the front-end JS script in the Block editor.
+	 *
+	 * @return void
+	 */
+	private function register_ece_data() {
+		$data_registry = Package::container()->get( AssetDataRegistry::class );
+
+		if ( $data_registry->exists( 'ece_data' ) ) {
+			return;
+		}
+
+		$ece_data = [
+			'button' => $this->get_button_settings(),
+		];
```

Comment:
We only need to pass the button settings to the Block editor so it gets all the styling defined in the WooPayments settings.